### DISCUSSION TITLE
Actividad 0 - Mariescurrena, Katerina

### DISCUSSION THREAD

**Discussion by KaterinaMariescurren** [frlp-utn-ingsoft/catedra#42] [Show and tell]
[https://github.com/KaterinaMariescurren/Ingenieria-y-Calidad-de-Software.git](url)

**Comment by RodrigoJacznik** [comment]
Hola
los commits podrían estar mejor, por ejemplo, commits que pertenecen a la "parte 4" quedaron con "parte 2".
La app funciona bien.
La actividad esta aprobada
